src/templates/static/legal/metatrader/risk_disclaimer.jsx

```diff
@@ -2,33 +2,36 @@ import React from 'react';
 
         <h2 data-anchor='external-forces'>{it.L('External forces')}</h2>
-        <p>{it.L('Your forex and contracts for differences transactions with [_1] are linked to an underlying financial instrument and you will be exposed to price, currency exchange, interest rate or other volatility of the market to which your financial instrument is related to, especially if the market is of a jurisdiction that is foreign to you. Forex and contracts for differences trading over the internet via an electronic trading platform is subject to data latency risk which may result in delays in order execution.', it.website_name)}</p>
+        <p>{it.L('The client\'s forex and contract-for-difference transactions with [_1] are linked to an underlying financial instrument, and the client will be exposed to price, currency exchange, interest rate, or other volatility of the market to which the client\'s financial instrument is related to, especially if the market is of a jurisdiction that is foreign to the client. Forex and contract-for-difference trading over the internet via an electronic trading platform is subject to data latency risk, which may result in delays in order execution.', it.website_name)}</p>
+
+        <h2 data-anchor='risk-of-cryptocurrency-cfd'>{it.L('Risk of cryptocurrency contracts for differences')}</h2>
+        <p>{it.L('Cryptocurrencies are volatile assets that are extremely vulnerable to market fluctuations. Trading cryptocurrency contracts for differences is a highly speculative investment because prices of cryptocurrencies can go up and down — potentially to a worthless (zero) value. Investing in cryptocurrency contracts for differences through leveraged accounts amplifies this underlying volatility risk, making the change in prices even more significant. When the client invests in a leveraged contract for differences related to cryptocurrencies, the client\'s profit or loss is based on the leveraged ratio that the client has set for the client\'s account. If the price movement is in the client\'s favour, returns generated are multiple times higher than the amount set aside from the client\'s account balance to open and maintain a transaction. However, if the price movement is against the client\'s favour, the client will suffer losses that are multiple times higher than the amount set aside from the client\'s account balance to open and maintain a transaction. The client should not trade in cryptocurrencies unless the client fully understands and is aware that trades and/or investments involving cryptocurrencies are complex and high risk in nature, and the client not only risks incurring significant losses over a short period of time but also risks losing all the client\'s invested capital.')}</p>
     </div>
```

Comment:
I can't find the rest of this part. The last two paragraphs of the Risk disclaimer file seem to be missing. Here are the two paragraphs I'm talking about: 

![image](https://user-images.githubusercontent.com/50129468/56977510-e65bd800-6ba7-11e9-8ba9-633f42db674c.png)

Reply:
They are in the other risk_disclaimer file, line 61. Are they in the wrong place?

Reply:
No, my mistake :-) Sorry! Everything is at its appropriate place.